### PR TITLE
inital Cart design

### PR DESCRIPTION
**Pull request by Dineshsoni02**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -157,6 +158,7 @@ function App() {
                 }
               />
               <Route path="/" element={<HomePage />} />
+              <Route path="/cart" element={<CartPage />} />
```
Comment (aashu0148) [inline src/App.js:161]: "/" vala route sabse last me aata hai so "/cart" vala route "/" vale se uper aayega

File: src/components/MerchantSidepanel/MerchantSidepanel.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { useState } from "react";
 import { NavLink } from "react-router-dom";
 import { Menu, X, Package, BarChart2 } from "react-feather";
 
-import styles from "./MerchantSidePanel.module.scss";
+import styles from "./MerchantSidepanel.module.scss";
```
Comment (aashu0148) [inline src/components/MerchantSidepanel/MerchantSidepanel.js]: ye change nhi karna tha

panel ka 'p' capital hi aayega

4 jagah check kar liye - MerchantSidePanel folder , MerchantSidePanel.js file , MerchantSidePanel.module.scss file , MerchantDashboard.js file

File: src/pages/Customer/Cart/CartPage.js [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+                    />
+                    <div className={styles.discountPrice}>-5%</div>
+                  </div>
+                  <div className={styles.aboutBox}>
+                    <div className={styles.productName}>
+                      Apples Lorem ipsum dolor sit amet.
+                    </div>
+                    <div className={styles.availableIn}>5kg</div>
+                    <div className={styles.productPrice}>₹ 30.9</div>
+                  </div>
```
Comment (aashu0148) [inline src/pages/Customer/Cart/CartPage.js]: line no 19-162 ye hard code nhi karte

isko map karke kiya ja sakta hai vo acha tarika hai

like make a array of products=[{...},{...},{...}]
then map through this array to print all these.

File: src/pages/Customer/Cart/CartPage.js [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+                      setQuantity((prev) => (prev > 1 ? prev - 1 : prev))
+                    }
+                  />
+                  <div className={styles.qtyValue}>{quantity}</div>
+                  <Plus
+                    className={styles.qtyInc}
+                    onClick={() => setQuantity((prev) => prev + 1)}
+                  />
+                </div>
+                <div className={styles.totalItem_price}>₹ 464.12</div>
```
Comment (aashu0148) [inline src/pages/Customer/Cart/CartPage.js]: na hi koi value hardcoded hogi

array jo banaega usme values hongi and vala se print karni hai

File: src/pages/Customer/Cart/CartPage.js [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+                <div className={styles.cartTotal_text}>Total Price</div>
+                <div className={styles.cartTotal_number}>₹ 123.123</div>
+              </div>
+            </div>
+            <div className={styles.totalPrice_bottomtext}>
+              Procced to Checkout for Payment details.
+            </div>
+          </div>
+          <div className={styles.checkoutBox}>
+            <button type="submit" className={styles.checkoutButton}>
```
Comment (aashu0148) [inline src/pages/Customer/Cart/CartPage.js]: type="submit" dene ka koi point nhi hai as ye sab form ke andar nhi hai

File: src/pages/Customer/Cart/CartPage.module.scss [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+      }
+    }
+    * {
+      user-select: none;
+    }
+    .leftInner {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      padding-top: 3%;
```
Comment (aashu0148) [inline src/pages/Customer/Cart/CartPage.module.scss]: padding , margin hum 'px' me rakh rahe hai '%' me nhi

File: src/pages/Merchant/MerchantDashboard/MerchantDashboard.js [hunk 7]
```diff
@@ -4,7 +4,7 @@ import PropTypes from "prop-types";
 
 import PrivateRoute from "components/PrivateRoute/PrivateRoute";
 import MerchantOverView from "./MerchantOverview/MerchantOverview";
-import MerchantSidePanel from "components/MerchantSidePanel/MerchantSidePanel";
+import MerchantSidePanel from "components/MerchantSidepanel/MerchantSidepanel";
```
Comment (aashu0148) [inline src/pages/Merchant/MerchantDashboard/MerchantDashboard.js]: no need to change that

File: src/pages/Customer/Cart/CartPage.js (hunk 3 above)
Comment (Dineshsoni02) [reply src/pages/Customer/Cart/CartPage.js:36]: to design to bnaya h api to iske baad hito add hogi
tbhi to data aayega

**Pull request closed without merging**